### ISSUE TITLE
Update Event Display to support new event categories (One-Time Events + Dauerveranstaltungen)

### ISSUE THREAD

**Issue by hauf-toni** [digitalfabrik/integreat-app#2935]
**description**

Municipalities have raised concerns about the current 28-day restriction on event durations in our event calendar. This limitation affects the posting of longer events. The backend team is proposing changes to the event creation process to accommodate multiple event lenghts, see [#3070
](https://github.com/digitalfabrik/integreat-cms/issues/3070)

We need to update the frontend to properly display these new event categories and improve the user experience when browsing the event calendar.

**Motivation**

User Autonomy: Allow CMS users the flexibility to create events that accurately reflect their municipality's offerings.
Improved Clarity: Enhance the event calendar's usability by properly categorizing and displaying events based on their duration.

**Proposed Solution**

Event Calendar Display:
- Modifying the listing logic to prevent long-term events from overshadowing one-day events.
- Implementing sorting or filtering options to balance the visibility of all event types.
- Visual Indicators like icons or labels to distinguish between one-time events and Dauerveranstaltungen.
- User Guidance, eg providing tooltips or help sections to explain the event categories to frontend users.

**user story**

As a user, I want to easily find and distinguish between one-time events and long-term events in the event calendar so that I can effectively plan my participation in municipal activities.

**Acceptance Criteria**

- Events are correctly categorized and displayed based on their type (one-time or Dauerveranstaltungen).
- Users can effortlessly navigate between different event types.
- Long-term events do not push one-day events to the back of the list.
- Clear visual cues differentiate event categories.
- The calendar maintains performance and responsiveness despite the additional categorization.

---

**Alternatives Considered**

- Keeping the old settings, which do not address the municipalities' needs.
- Remove restrictions entirely and allow any event duration, potentially cluttering the event calendar.
- Implementing a review system for events exceeding a certain duration, adding administrative overhead.

---

**Dependencies**

- Coordination with the backend team to ensure frontend changes align with the updated data structures and APIs: https://github.com/digitalfabrik/integreat-cms/issues/3070
- Translation and localization updates for any new text or labels introduced.

**Related issues**

#2986, #2808, #2707, #2112
~~Blocked by: https://github.com/digitalfabrik/integreat-cms/issues/3070~~

**Comment by steffenkleinle** [comment]
@hauf-toni will you provide a design for this or should we just try to do this ourselves?

**Comment by hauf-toni** [comment]
@steffenkleinle I will provide a design for this

**Comment by JoeyStk** [comment]
@LeandraH / @steffenkleinle we're unsure what you still need from our side to unblock this issue. Could you please tell us what you need from us here? :)

**Comment by steffenkleinle** [comment]
> [@LeandraH](https://github.com/LeandraH) / [@steffenkleinle](https://github.com/steffenkleinle) we're unsure what you still need from our side to unblock this issue. Could you please tell us what you need from us here? :)

I guess we just did not realize that the corresponding CMS issue is already done. So I guess nothing and this is unblocked?

**Comment by JoeyStk** [comment]
No worries. We might have also forgotten to tell you about it. But juhu, then this is unblocked 🙌

**Comment by steffenkleinle** [comment]
@ulyanahoy-cloud is this still planned or already done with #4321? I am not sure whether users actually need to clearly differentiate between single and multi-day events.
